wire-ios-request-strategy/Sources/E2EIdentity/EnrollE2eICertificateUseCase.swift

```diff
@@ -47,7 +48,11 @@ public final class EnrollE2eICertificateUseCase: EnrollE2eICertificateUseCaseInt
                        userHandle: String,
                        team: UUID,
                        authenticate: OAuthBlock) async throws {
-        try await e2eiRepository.fetchTrustAnchor()
+        do {
+            try await e2eiRepository.fetchTrustAnchor()
```

Comment:
Is it optional to do this?

Reply:
Now It throws an error. I talked with Mojtaba and he suggested to log an error for now until it gets fixed on the CC side.

Reply:
Do you want to add a TODO here?

Reply:
I don't have a ticket for TODO and don't know when CC will fix it.